app/controllers/hub/organizations_controller.rb

```diff
@@ -0,0 +1,48 @@
+module Hub
+  class OrganizationsController < ApplicationController
+    include AccessControllable
+    before_action :require_sign_in
+    load_and_authorize_resource :vita_partner, parent: false
+
+    layout "admin"
+
+    def new
+      @coalitions = Coalition.all
+    end
+
+    def create
+      @coalitions = Coalition.all
```

Comment:
Is @coalitions only used in the case where the organization.save fails and rerenders new? In that case, might be better to move it into the block that renders :new vs always calling it!